tests/test_app.py

```diff
@@ -1,14 +1,28 @@
+import mock
 import unittest
+
 from ubuntucleaner.main import UbuntuCleanerWindow
 
 
-class TestApp(unittest.TestCase):
+class TestUbuntuCleanerWindow(unittest.TestCase):
```

Comment:
Simply renamed to follow the convention of matching the class name we're testing